addon/components/bs-modal.js

```diff
@@ -90,17 +84,6 @@ export default class Modal extends Component {
   @tracked
   showModal = this.open && (!this._fade || isFastBoot(this));
```

Comment:
Seems unused now?

But looking at this, this could also explain the FastBoot issue, as this previous property would make the modal be visible in FastBoot without having to rely on the `handleVisibilityChanges` method, which probably does not even run in FastBoot?

We might apply a similar initializer to the `state` property now (instead of setting to `closed` and again relying on `handleVisibilityChanges()`? 🤔

Reply:
Yes. That's also what I guess.

ember-render-helpers should be executed in FastBoot. Helpers are executed in FastBoot. And I haven't noticed any FastBoot logic in ember-render-helpers itself.

I suspect a FastBoot guard in the `show` method to be to strict and prevent needed work. But need to dig deeper into that one to verify.